tests/views/test_message.py

```diff
@@ -129,3 +129,35 @@ def test_empty_subject_and_body_rejected(self):
 
         self.assertIn("Please enter a subject".encode(), response.data)
         self.assertIn("Please enter a message".encode(), response.data)
+
+    @requests_mock.mock()
+    def test_form_submit_with_valid_data(self, mock_request):
+        mock_request.post(f'{app.config["BACKSTAGE_API_URL"]}/v1/secure-message/send-message', status_code=201)
+        mock_request.get(f'{app.config["BACKSTAGE_API_URL"]}/v1/secure-message/messages', json={}, status_code=200)
+
+        with app.app_context():
+            message_form = {'body': "TEST BODY",
+                            'subject': "TEST SUBJECT"}
+
+            response = self.app.post(self.create_message_url, data=message_form, follow_redirects=True)
+
+            self.assertIn("Message sent.".encode(), response.data)
+
+    @requests_mock.mock()
+    def test_form_submitted_with_api_error(self, mock_request):
+        mock_request.post(f'{app.config["BACKSTAGE_API_URL"]}/v1/secure-message/send-message', status_code=500)
+
+        with app.app_context():
+            message_form = {'body': "TEST BODY",
```

Comment:
In the assertIn block below, test that the body and subject get repopulated after failed submission